src/util/argstr.rs

```diff
@@ -102,6 +106,7 @@ impl<'a> ArgStr<'a> {
         }
     }
 
+    #[allow(dead_code)]
     pub(crate) fn as_raw_bytes(&self) -> &[u8] {
```

Comment:
This method is used in some debugging code, but `ArgStr` is also included in the output. Should I remove it or leave the `#[allow]`?

Reply:
I believe it's fine. I'll be getting to eradicating these warnings tomorrow anyway.

Reply:
I'm not sure I understand what you mean?

Reply:
@kbknapp This function is only used in debug mode. In release, it will trigger `function is never used` warning.

Reply:
Ah ok, I read that totally wrong. 

A better way to handle that might be `#[cfg(debug_assertions)]` which should only build that function in debug mode. But I haven't tested that in some time.

Reply:
@CreepySkeleton is right. I could have worded that better.